examples.md

```diff
@@ -1,3 +1,17 @@
+<!-- vim-markdown-toc GFM -->
```

Comment:
What does this do? "<!-- vim-markdown-toc GFM -->"

I like having a ToC. It just fits a separate PR.

Reply:
It's just a VIM plugin which automatically update TOC.
OK. I will remove it. I don't want to spend that time discussing that thing.